### PR TITLE
Build playlist management page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
- Implemented functionality of creating a new playlist
- delete playlist
- add a video to a playlist
- remove the video from the playlist
- different page for playlist videos

Preview Link - 
https://deploy-preview-3--learncomputerscience.netlify.app/

Do signup to test the above functionalities.

### CODE REVIEW

File: src/components/category card/CategoryCard.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+              payload: category.categoryName,
+            })
+          }
+        >
+          <img className="card-img" src={category.image} alt="category" />
+        </Link>
         <div className="card-text-overlay">
-          <div className="card-header-title bold-font-weight">
+          <div className="card-header-title bold-font-weight category-style ">
             {" "}
```
Comment (hrishikeshmane) [inline src/components/category card/CategoryCard.js]: remove formatted empty spaces

File: src/components/model/add-to-playlist-modal/AddToPlayListModel.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                  key={playlist._id}
+                  htmlFor=""
+                  className="add-playlist-model-list-item"
+                >
+                  <input
+                    type="checkbox"
+                    id={playlist._id}
+                    // checked={}
+                    onChange={(e) =>
+                      // console.log(isVideoExistInPlayList(videoId,playlist))
```
Comment (hrishikeshmane) [inline src/components/model/add-to-playlist-modal/AddToPlayListModel.js]: remove commented code.

File: src/components/model/add-to-playlist-modal/addtoplaylistmodel.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  align-items: center;
+  justify-content: center;
+}
+
+.add-playlist-model-content {
+  background-color: var(--white);
+  display: flex;
+  flex-direction: column;
+  width: 20%;
+  /* height: 13rem; */
```
Comment (hrishikeshmane) [inline src/components/model/add-to-playlist-modal/addtoplaylistmodel.css]: here aswell

File: src/pages/playlist/PlayList.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                          )
+                        }
+                      ></i>
+                      <button className="no-style-btn">Watch Later</button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            ))
+          : ""}
```
Comment (hrishikeshmane) [inline src/pages/playlist/PlayList.js]: suggestion:  use logical AND (&&) instead of  ternary (? :)

**Review by hrishikeshmane — APPROVED** [review]
Rest looks good

**Review by anupkgurung — APPROVED** [review]
clean code,
nice code structure

File: src/components/category card/CategoryCard.js (hunk 1 above)
Comment (ishitakeshawani) [reply src/components/category card/CategoryCard.js]: done

File: src/components/model/add-to-playlist-modal/AddToPlayListModel.js (hunk 2 above)
Comment (ishitakeshawani) [reply src/components/model/add-to-playlist-modal/AddToPlayListModel.js]: done

File: src/components/model/add-to-playlist-modal/addtoplaylistmodel.css (hunk 3 above)
Comment (ishitakeshawani) [reply src/components/model/add-to-playlist-modal/addtoplaylistmodel.css]: done

File: src/pages/playlist/PlayList.js (hunk 4 above)
Comment (ishitakeshawani) [reply src/pages/playlist/PlayList.js]: done

**Pull request merged**